test/integration/suite_test.go

```diff
@@ -85,7 +85,7 @@ var _ = BeforeSuite(func() {
 		CRDDirectoryPaths: []string{
 			filepath.Join("../../", "config", "crd", "bases"),
 			filepath.Join("../../", "config", "gateway-api", "crd", "standard"),
-			filepath.Join("../../", "config", "cert-manager", "crd", "v1.7.1"),
+			filepath.Join("../../", "config", "cert-manager", "crd", "v1.12.2"),
```

Comment:
@david-martin Do you think there is any advantage to keeping version of these crds rather than just creating a "latest" directory and keeping that up to date?

Reply:
I can't think of a reason to keep older versions.

Reply:
Decided to rename the directory so we should never have to update these tests again